Makefile

```diff
@@ -75,6 +75,8 @@ TESTS ?=
 CPP_TEST_SOURCES := test/native/testRunner.cpp $(shell find test/native -name '*Test.cpp')
 CPP_TEST_HEADER := test/native/testRunner.hpp
 CPP_TEST_INCLUDES := -Isrc -Itest/native
+CPP_TEST_LIB_SOURCES := test/native/testRunner.cpp $(shell find test/native/libs \( -name "*.c" -o -name "*.cpp" \))
```

Comment:
There are also other file types in that directory: `.s` and `.ld`. Why not just include all files? Since there are no subdirectories, you can just use built-in $(wildcard) instead of shell find.

Reply:
Also, why does it depend on `testRunner.cpp`? This is only for unit tests.